docs/src/usage.md

```diff
@@ -1,12 +1,93 @@
-# Usage
+# Using Kani
 
 At present, Kani can used in two ways:
- * [On a single file](./kani-single-file.md) with the `kani` command.
- * [On a package](./cargo-kani.md) with the `cargo-kani` command.
 
-Running [Kani on a single file](./kani-single-file.md) is quite useful for small
-examples or projects that don't use `cargo`.
+ * [On a single crate](#usage-on-a-single-crate) with the `kani` command.
+ * [On a Cargo package](#usage-on-a-package) with the `cargo kani` command.
 
-However, if you plan to integrate Kani in your projects, the recommended
-approach is to use [Kani on a package](./cargo-kani.md) because of its ability
-to handle external dependencies.
+If you plan to integrate Kani in your projects, the recommended approach is to use `cargo kani`.
+But `kani` is useful for small examples/tests.
+
+## Usage on a package
+
+Kani is also integrated with `cargo` and can be invoked from a package as follows:
+
+```bash
+cargo kani [OPTIONS]
+```
+
+This works like `cargo test` except that it will run all proof harnesses instead of running all test harnesses.
+
+## Common command line flags
+
+Common to both `kani` and `cargo kani` are many command-line flags:
+
+ * `--visualize`: Generates an HTML report showing coverage information and providing traces (i.e., counterexamples) for each failure found by Kani.
+
+ * `--tests`: Build in "[test mode](https://doc.rust-lang.org/rustc/tests/index.html)", i.e. with `cfg(test)` set and `dev-dependencies` available (when using `cargo kani`).
+
+ * `--harness <name>`: By default, Kani checks all proof harnesses it finds.
+   You can switch to checking a single harness using this flag.
+
+ * `--default-unwind <n>`: Set a default global upper [loop unwinding](./tutorial-loop-unwinding.md) bound for proof harnesses.
+   This can force termination when CBMC tries to unwind loops indefinitely.
+
+Run `cargo kani --help` to see a complete list of arguments.
+
+## Usage on a single crate
+
+For small examples or initial learning, it's very common to run Kani on just one source file.
+The command line format for invoking Kani directly is the following:
+
+```
+kani filename.rs [OPTIONS]
+```
+
+This will build `filename.rs` and run all proof harnesses found within.
+
+## Configuration in `Cargo.toml`
```

Comment:
This section seems to now address https://github.com/model-checking/kani/issues/1390?

Reply:
Yeah, will tag in PR